x/evm/migrations/migrate_deliver_tx_gas_limit_test.go

```diff
@@ -0,0 +1,26 @@
+package migrations_test
+
+import (
+	"testing"
+
+	testkeeper "github.com/sei-protocol/sei-chain/testutil/keeper"
+	"github.com/sei-protocol/sei-chain/x/evm/migrations"
+	"github.com/sei-protocol/sei-chain/x/evm/types"
+	"github.com/stretchr/testify/require"
+	tmtypes "github.com/tendermint/tendermint/proto/tendermint/types"
+)
+
+func TestMigrateDeliverTxHookWasmGasLimitParam(t *testing.T) {
+	k := testkeeper.EVMTestApp.EvmKeeper
+	ctx := testkeeper.EVMTestApp.NewContext(false, tmtypes.Header{})
+
+	err := migrations.MigrateDeliverTxHookWasmGasLimitParam(ctx, &k)
+
+	require.NoError(t, err)
+
+	require.NotPanics(t, func() { k.GetParams(ctx) })
+
+	keeperParams := k.GetParams(ctx)
+
+	require.Equal(t, keeperParams.GetDeliverTxHookWasmGasLimit(), types.DefaultParams().DeliverTxHookWasmGasLimit)
```

Comment:
For good measure we can start by setting some other value for a non new param and verify that it doesn't get overwritten

Reply:
yeah i relized that too. working on it